source/system-administrators/upgrade/upgrading-to-craftercms-3-0.rst

```diff
@@ -41,8 +41,15 @@ Authoring
 Export the 2.5.x Site
 ^^^^^^^^^^^^^^^^^^^^^
 
-Export the site's Studio config (``cstudio/config/sites/{siteName}``) and the site's content (``wem_projects/{siteName}/{siteName}/work-area`` folders
-from Alfresco or from your local 2.5.x repository. Then, copy these folders into a location in the system with Crafter 3.0.x installed.
+#. Export the site's Studio config (``cstudio/config/sites/{siteName}``) and the site's content (``wem_projects/{siteName}/{siteName}/work-area``
+   from your Crafter 2.5 installation:
+
+   - If your Crafter 2.5 uses Alfresco as the repository, login to Alfresco Share and download this folders as zips from the
```

Comment:
Change `this` to `the` so it reads like: `login to Alfresco Share and download the folders as zips ...`